dcor/_energy.py

```diff
@@ -4,9 +4,14 @@
 
 import numpy as np
 
+import enum
+
 from . import distances
 from ._utils import _transform_to_2d
 
+class EstimationStatistic(enum.Enum):
+    USTATISTIC = 'u'
```

Comment:
I think that it is more aesthetically pleasing to use an underscore: `U_STATISTIC` instead of `USTATISTIC`. Also, I thing that the string should also be `"U_STATISTIC"` although I am not sure if `"U"` should also be accepted.